src/secureCreatives.js

```diff
@@ -190,7 +193,7 @@ function resizeRemoteCreative({ adId, adUnitCode, width, height }) {
     let element = getElementByAdUnit(elmType + ':not([style*="display: none"])');
     if (element) {
       let elementStyle = element.style;
-      elementStyle.width = width + 'px';
+      elementStyle.width = width ? width + 'px' : '100%';
```

Comment:
Out of curiosity - would there be scenarios where width is undefined but height is still there?  Is there a reason to not have a backup value for height as well?

Reply:
The scenario actually existed, it was into prebid-universal creative. I am linking one of the lines that was doing exactly that, setting the height but not the width: 

https://github.com/prebid/prebid-universal-creative/pull/150/files#diff-8938ddbc8ffb985e7aeb3744e592068ea58501c022a222cb7507aabef70f8080L296 

So, a change was needed to fix this scenario: when a native ad is rendered into a GAM _regular_ creative. 

You may know that native ads can be rendered into GAM Native Creatives, that have the `fluid` size. Fluid here means that the ad will try to expand horizontally to take the size of its container. 

When you try to do that into a regular GAM creative, this is more difficult. Say we're going to render an ad in a `1x1` regular creative. In this scenario, what would happen is that a 1 pixel-large iframe is created, the ad is rendered, but since the width is 1px, it will very tall. Later, the PUC calls `resizeNativeHeight`, that tries to fix the height of the container, leaving the width ad 1px. 

With the change that was added in prebid-universal-creative there are a few things that will happen: 
1. first, the ad's width will be expanded to the size of the container. This is done even if it's a safeframe. 
2. Then, we request the ad resize, but we pass the width too. 

Other scenarios were all passing the width, so I didn't add a default here.